modules/environment-roles/templates/grafana_monitoring_policy.json.tpl

```diff
@@ -0,0 +1,28 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Action": [
+        "logs:DescribeQueries",
+        "tag:GetResources",
+        "ec2:DescribeInstances",
+        "logs:DescribeLogGroups",
+        "cloudwatch:GetMetricData",
+        "logs:DescribeLogStreams",
+        "ec2:DescribeTags",
+        "ec2:DescribeRegions",
+        "logs:StartQuery",
+        "logs:StopQuery",
+        "cloudwatch:GetMetricStatistics",
+        "cloudwatch:ListMetrics",
+        "logs:GetQueryResults",
+        "cloudwatch:DescribeAlarmsForMetric",
+        "logs:GetLogEvents",
+        "logs:FilterLogEvents"
```

Comment:
Really minor, but could you sort this list so it's easier to scan the permissions?

Reply:
Fair point, cut and paste...

Reply:
Ah, did you copy the policy I created in the alpha prototype? 😳